### ISSUE TITLE
 shows wrong date(-1 day) reading data from db

### ISSUE THREAD

**Issue by sergeynilov** [mariomka/vue-datetime#70]
## Versions
- vue: 2.5.7
- vue-datetime: 1.0.0-beta.3
- luxon: ^1.2.1

### Description:

Hello ,
In  my  laravel 5.6 / vue.js/ Bootstrap4 I use
vue-datetime  and I found that when I open editor with current value from db
vue-datetime shows wrong date(-1 day)
so if in data
task.date_start= 2018-06-02

having definition :
```
	<datetime
		type="date"
		name="date_start"
		id="date_start"
		v-model="task.date_start"
		:format="{ year: 'numeric', month: 'long', day: 'numeric' }"
		:week-start="1"
		auto
		:value-zone="timezone"
		class="form-control"
	></datetime>
```

 I see that task.date_start has value = ‘2018-06-01T00:00:00.000Z’ and
```
 timezone::Europe/Kiev
```

 Is it some error  in my config ?

Thanks!

**Comment by franciscobarrerarodriguez** [comment]
Did you solve this?, I have the same issue.

**Comment by sergeynilov** [comment]
No, I did not find how to salve it. I suppose that could be base 0/1 issue.  
I waite for feedback of the developers of vue-datetime...

**Comment by arturs-buls** [comment]
I guess it is same issue as #47 . It has even pull request in it, but no sign of devs.... no commits from February.... Not loosing hope yet, but looks like there are no maintainers for this repo.
As temp solution for this I am adding 1 day on component creation 
`DateTime.fromISO(data.repeatUntil).plus({ days: 1 }).toISODate();`

**Comment by sergeynilov** [comment]
That is a pity, as this component seemed very useful...
If there is some alternative to this component ? bootstrap 4.1 compatible?

**Comment by mariomka** [comment]
This should be resolved thanks to #61.
